### DISCUSSION TITLE
狗狗币钱包私钥怎么导入

### DISCUSSION THREAD

**Discussion by yp7622266** [dogecoin/dogecoin#3711] [General]
狗狗币私钥怎么导入，我尝试在电脑导入
![50a9f2a29ff3b4b307d783065d9aa2b](https://github.com/user-attachments/assets/b706084c-ab62-4a0a-b1cd-808a17455e79)
显示失败。   
求详细导入私钥流程，本人不太懂电脑，麻烦发详细的步骤

**Comment by 0foxes** [comment]
诈骗，不要信楼上

请问你有私钥还是wallet.dat？
